content/en/news/releases/1.17.x/announcing-1.17.7/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing Istio 1.17.7
+linktitle: 1.17.7
+subtitle: Patch Release
+description: Istio 1.17.7 patch release.
+publishdate: 2023-10-10
+release: 1.17.7
+---
+
+This release fixes the security vulnerabilities described in our Oct 10th post, [`ISTIO-SECURITY-2023-004`](/news/security/istio-security-2023-004).
+
+This release note describes what’s different between Istio 1.17.6 and 1.17.7.
+
+{{< relnote >}}
+
+## Security update
+
+- __[`CVE-2023-48887`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

Comment:
```suggestion
- __[`CVE-2023-44487`](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)__: (CVSS Score 7.5, High): HTTP/2 denial of service
```

Reply:
Thanks Daniel, I've resolved all of these